packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7905,9 +7912,14 @@ export default class Meeting extends StatelessWebexPlugin {
     // https://github.com/sinonjs/fake-timers/issues/321
     const now = new Date().getTime();
 
+    const permissionExpiryTimestamp = permissionTokenExpValue;
```

Comment:
these 2 new consts are not used for anything apart from return, so why not just do:
```
return {permissionTokenTimeLeft, permissionExpiryTimestamp: permissionTokenExpValue, currentTime: now};
```
in line 7922 and not add anything here?